packages/gatsby/package.json

```diff
@@ -59,8 +59,8 @@
     "lint": "run-s lint:prettier lint:eslint",
     "lint:eslint": "eslint . --cache --cache-location '../../eslintcache/' --format stylish",
     "lint:prettier": "prettier --check \"{src,test,scripts}/**/*.ts\"",
-    "test": "jest",
-    "test:watch": "jest --watch"
+    "test": "yarn ts-node scripts/pretest.ts && yarn jest",
+    "test:watch": "yarn ts-node scripts/pretest.ts && yarn jest --watch"
```

Comment:
Why are we doing this for gatsby in particular?

Reply:
Because it needs files other than what you get by building the `cjs` and `esm` directories and building types, specifically the ones created by https://github.com/getsentry/sentry-javascript/pull/4928.

Reply:
UPDATE: Added a note to the PR description.